packages/contracts/contracts/core/DAO.sol

```diff
@@ -207,14 +207,14 @@ contract DAO is
         override
         auth(address(this), SET_SIGNATURE_VALIDATOR_PERMISSION_ID)
     {
-        signatureValidator = ERC1271(_signatureValidator);
+        signatureValidator = IERC1271(_signatureValidator);
```

Comment:
looking at the code, 
i think we are missing some parts:
- we need to setSignatureValidator() in the init func.
- we need an event related to setSignatureValidator.

Reply:
Agreed